### ISSUE TITLE
Add blog to the site.

### ISSUE THREAD

**Issue by naveenkumarsangi** [NIT-dgp/nit-dgp.github.io#18]
This issue typically requires the following
- [ ] Make a post layout in `_layouts/post.html`
- [ ] Add decent CSS.
- [ ] Add a sample **Hello World** post in `posts/` directory using **post** layout.

**Comment by monsij** [comment]
@nkprince007  : Can I work on this issue and consequently get assigned?

**Comment by naveenkumarsangi** [comment]
Yeah sure. But you should be adding stuff related to each project in a separate pull request.

**Comment by monsij** [comment]
Yep sure!
